src/coreclr/debug/createdump/crashreportwriter.cpp

```diff
@@ -51,20 +51,19 @@ CrashReportWriter::WriteCrashReport(const std::string& dumpFileName)
     }
 }
 
-#ifdef __APPLE__
-
 void
 CrashReportWriter::WriteCrashReport()
 {
-    const char* exceptionType = nullptr;
     OpenObject("payload");
-    WriteValue("protocol_version", "0.0.7");
+    WriteValue("protocol_version", "1.0.0");
```

Comment:
When you say protocol 1, it's usually we are not making breaking changes until 2. Is this what we are intending? I know this is not a very public protocol, but I think it's worth to consider nonetheless.

Reply:
For the mono analyzer as long as it is greater than 0.0.6 it should be fine. Kyle mentioned just going to 1.0.0 so I thought I start both MacOS/Linux there.